package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ff-nodejs-server-sdk",
-  "version": "1.0.1",
+  "version": "1.1.1",
```

Comment:
Since we introduced a breaking change by completely changing the API of the SDK to be asynchronous across the board, this really aught to be at `2.0.0` by now if we're sticking to SemVer

Reply:
By changing the returns of all the variation function to be promises, everything is broken. Code written for `v1.0.0` will not work with `v1.1.0`

Reply:
Also, this should only be `v1.1.1` if it _only_ fixes bugs in `v1.1.0`, if there are any breaking changes (e.g. changes in function signatures/return types), then it should be `v2.0.0` and any new features or changes that do not break the existing signature should be `v1.1.0`

Reply:
The new thing is we introduce waitForInit so I think it is a requested feature from customer :), So that is the reason for 1.1.0 version